src/productWijzigen.php

```diff
@@ -0,0 +1,128 @@
+include "functions/userFunctions.php";
+
+session_start();
+
+/*if(!isset($_SESSION['login'])){
+  header('location: index.php');
+  return;
+};*/
+
+
+if (isset($_POST['submit'])) {
+	$naam= $_POST['naam'];
+  	$prijs= $_POST['prijs'];
+  	$beschrijving= $_POST['beschrijving'];
+  	$categorie= $_POST['categorie'];
+  	$entime = date('Y-m-d H:i:s', strtotime('+12 hours'));
+  	$upload_dir= $_SERVER['DOCUMENT_ROOT']."/2dekans-veilingen-A/public/img/";
+  	$file_name= $_FILES['file']['name'];
+  	$file_tmp= $_FILES['file']['tmp_name'];
+  if (filesize( $file_name ) < 0){
+  echo "Error";
+  }
+  if((empty($_POST['file']))){
+    if (!(move_uploaded_file($file_tmp,$upload_dir.$file_name))) {
+      echo "Error, kon de foto niet verplaatsen.";
+    }
+  }
+    if(modifyProduct($mysqli, $naam, $beschrijving, $prijs, $categorie, $file_name)){
+    header('location: index.php');
+    }else{
+	echo "Error product wijzigen.";
+    }
+
+}else{
+
+
+  $productID = $_GET['gekozenProduct'];
+  foreach(getProduct($mysqli, $productID) as $row) {
+
+
+echo '<div>
+    <div class="flex justify-start items-start">
+      <a href="index.php" class="btn btn-ghost normal-case text-xl text-black">2dekans veilingen</a> 
+    </div>
+    <form class="form-control h-full flex items-center justify-center" action="productWijzigen.php" method="post" enctype="multipart/form-data">
+      <div class="card w-full max-w-lg shadow-2xl bg-white p-8 mx-auto justify-center items-center">
+        <h2 class="text-black text-2xl mb-4">Modify Product</h2>
+        <div class="flex flex-col gap-2">  
+          <div class="flex flex-row gap-2"> 
+            <div class="flex flex-col w-full"> 
+              <label class="label text-black">Product</label>
+              <input type="text" name="naam" placeholder="Product" value = "'.$row["naam"].'" class="input input-bordered w-full max-w-md text-black bg-white" required  />
+            </div>
+            <div class="flex flex-col w-full"> 
+              <label class="label text-black">Minimale prijs</label>
+              <input type="text" name="prijs" placeholder="Minimale prijs" value = "'.$row["prijs"].'" class="input input-bordered w-full max-w-md text-black bg-white" required />
+            </div>
+          </div>
+          <div class="flex flex-row gap-2">
+            <div class="flex flex-col w-full"> 
+              <label class="label text-black">Beschrijving</label>
+              <textarea class="textarea textarea-bordered h-24  text-black bg-white" name="beschrijving" placeholder="Beschrijving"  required>'.$row["beschrijving"].'</textarea>
+            </div>
+          </div>
+          <div class="flex flex-row gap-2">
+            <div class="flex flex-col w-full"> 
+              <label class="label text-black">Productfoto</label>
+              <input type="file" name="file" class="file-input file-input-bordered bg-white text-black" value = "'.$row["foto"].'" required />
```

Comment:
Niet required want als ik de zelfde foto wil behouden moet dat kunnen. zorg hiervoor dat als deze empty is het gewoon de profielfoto pakt van de database. voor te zien hoe dit werkt kijk naar aanpassenGebruikers.php